server/config/service.go

```diff
@@ -247,8 +246,15 @@ func (s *service) getMattermostLicense(log utils.Logger) *model.License {
 }
 
 func (s *service) Reconfigure(newStoredConfig StoredConfig, verbose bool, services ...Configurable) error {
-	// Use the old settings to log the processing of the new config.
-	clone, log, err := s.newInitializedConfig(newStoredConfig)
```

Comment:
This fixes another issue: Even if `verbose` was `false` `newInitializedConfig` would still log it's messages on every configuration change.